app/i18n/en-US/chatbot.json

```diff
@@ -314,26 +314,68 @@
+  "Cloudbot Media Share Preferences": "Cloudbot Media Share Preferences",
+  "Cloudbot Heist Preferences": "Cloudbot Heist Preferences",
+  "Cloudbot Gamble Preferences": "Cloudbot Gamble Preferences",
+  "Enabling Song Request": "Enabling Song Request",
+  "You don't have any Commands": "You don't have any Commands",
+  "Click Add Command to get started":"Click Add Command to get started",
+  "Go to settings to customize your loyalty system": "Go to settings to customize your loyalty system",
+  "You don't have any Quotes": "You don't have any Quotes",
+  "Click Add Quote to get started or use !addquote <text> in chat": "Click Add Quote to get started or use !addquote <text> in chat",
+  "You don't have any Timers": "You don't have any Timers",
+  "Click Add Timer to get started": "Click Add Timer to get started",
+  "You don't have any queued users yet": "You don't have any queued users yet",
+  "You don't have any pending users yet": "You don't have any pending users yet",
+  "Open the queue to get started": "Open the queue to get started",
+  "You don't have any Blacklisted Links yet.": "You don't have any Blacklisted Links yet.",
+  "You don't have any Whitelisted Links yet.": "You don't have any Whitelisted Links yet.",
+  "Click Add Link to get started.": "Click Add Link to get started.",
+  "You don't have any Blacklisted Words/Phrases": "You don't have any Blacklisted Words/Phrases",
+  "Click Add Word/Phrase to get started": "Click Add Word/Phrase to get started",
+  "Emote Protection": "Emote Protection",
+  "Removes messages containing excessive an amount of emotes.": "Removes messages containing excessive an amount of emotes.",
+  "Paragraph Protection": "Paragraph Protection",
+  "Prevents viewers from sending overly lengthy messages.": "Prevents viewers from sending overly lengthy messages.",
+  "Desktop Streamlabs Chatbot": "Desktop Streamlabs Chatbot",
+  "Instructions": "Instructions",
+  "In order to import your data from our Desktop Chatbot you will need to follow these steps:": "In order to import your data from our Desktop Chatbot you will need to follow these steps:",
+  "Open up your Streamlabs Chatbot": "Open up your Streamlabs Chatbot",
+  "Click on": "Click on",
+  "to open the connections window": "to open the connections window",
+  "Click on the Streamlabs": "Click on the Streamlabs",
+  "Make sure your Streamlabs Account is connected": "Make sure your Streamlabs Account is connected",
+  "Select which data you wish to export": "Select which data you wish to export",
+  "Click Export and wait for the bar to fill completely": "Click Export and wait for the bar to fill completely",
+  "Ok": "Ok",
+  "Yes": "Yes",
+  "No": "No",
+  "Reset": "Reset",
+  "Minimum Amount of Emotes": "Minimum Amount of Emotes",
+  "Maximum Amount of Emotes": "Maximum Amount of Emotes",
+  "1-25": "1-25",
+  "26-50": "26-50",
+  "51-75": "51-75",
+  "76-98": "76-98",
+  "99-100": "99-100",
+  "Tie Message": "Tie Message",
+  "#": "#"
```

Comment:
Numbers and special characters must not be a part of translation

Reply:
Just use these strings without `$t` wrapper

Reply:
Done